nats-server/serverz.yml

```diff
@@ -0,0 +1,53 @@
+type: json
+data:
+  url: /streaming/serverz
+  server: ${NATS_SERVER_HOSTPORT}
+
+observation:
+  - name: serverz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: streaming.clients
+        source: clients
+        type: long_gauge
+        label: clients
+        description: The number of clients
+
+      - name: streaming.subscriptions
+        source: subscriptions
+        type: long_gauge
+        label: subscriptions
+        description: The number of subscriptions
+
+      - name: streaming.channels
+        source: channels
+        type: long_gauge
+        label: channels
+        description: The number of channels
+
+      - name: streaming.messages
+        source: total_msgs
+        type: long_gauge
```

Comment:
Counter or gauge? Same for other metrics here.

Reply:
Updated to counter